static/scripts/audit-report/audit.ts

```diff
@@ -315,244 +364,93 @@ class QueueSet {
 const updateQueue = new SmartQueue();
 const rpcQueue = new QueueSet();
 
-async function commentFetcher() {
-  if (isComment) {
-    const commentIntervalID = setInterval(async () => {
-      clearInterval(commentIntervalID);
-      try {
-        if (issueList.length !== 0) {
-          const octokit = new Octokit({
-            auth: GITHUB_PAT,
-            throttle: {
-              onRateLimit: (retryAfter, options) => {
-                return primaryRateLimitHandler(retryAfter, options as RateLimitOptions);
-              },
-              onSecondaryRateLimit: (retryAfter, options) => {
-                return secondaryRateLimitHandler(retryAfter, options as RateLimitOptions);
-              },
-            },
-          });
-
-          const [owner, repo] = parseRepoUrl(issueList[0].html_url);
-
-          const { data } = await octokit.rest.issues.listComments({
-            owner,
-            repo,
-            issue_number: issueList[0].number,
-            per_page: offset,
-            page: commentPageNumber,
-          });
-
-          await fetchComment(owner, repo, data);
-        } else {
-          isComment = false;
-          finishedQueue.mutate("isComment", true);
-        }
-      } catch (error) {
-        console.error(error);
-        finishedQueue.raise();
-        issueList.shift();
-        if (issueList.length > 0) {
-          await commentFetcher();
-        } else {
-          isComment = false;
-          finishedQueue.mutate("isComment", true);
-        }
-      }
-    }, GIT_INTERVAL);
-  }
-}
-
-async function fetchComment(owner: string, repo: string, data: ListCommentsDataItem[]) {
-  let isFound = false;
-  if (data.length === 0) {
-    commentPageNumber = 1;
-    issueList.shift();
-    if (issueList.length > 0) {
-      await commentFetcher();
-    } else {
-      isComment = false;
-      finishedQueue.mutate("isComment", true);
-    }
-  } else {
-    isFound = await processComments(owner, repo, data);
-
-    if (isFound) {
-      commentPageNumber = 1;
-      issueList.shift();
-    } else {
-      commentPageNumber++;
-    }
-
-    if (issueList.length > 0) {
-      commentFetcher().catch((error) => console.error(error));
+async function getPermitsForRepo(owner: string, repo: string) {
+  const permitList: Permit[] = [];
+  try {
+    const { data: gitData } = await octokit.rest.repos.get({
```

Comment:
We should take the sign in with GitHub implementation I made on work.ubq.fi and authenticate as the user for increased rate limits. 

How is authentication being handled now?

Reply:
There is no auth in the current implementation, I think. We could open a separate issue for this.